spark-common/src/main/js/src/spark-common-templates.js

```diff
@@ -1,12 +1,34 @@
+const escapeHtml = function(str) {
+    if (!str) {
+        return null;
+    }
+    return str.replace(/</g, '&lt;')
```

Comment:
Question: Do we need to escape the ampersand character here as well?

I'm not quite sure how to handle this correctly in the String templates.
This might be interesting: https://mrcoles.com/blog/how-use-amersands-html-encode/

Reply:
good question, I've thought about a bit but my understanding is that unescaped '&' cannot really cause XSS or similar problems in any context, at max some broken html entities in some edge cases

if it's not escaped, the escaping is stable, ie it can be run on already escaped input and nothing changes

Not escaping it would not cause any potential security issues and would not be a breaking change for apps giving already escaped values as input (though I guess no app does that)

But yes, probably would be better to escape that too, and make sure that internally no double escaping happens (like the case you pointed out)